src/components/Books.js

```diff
@@ -1,14 +1,27 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/Books/Books';
+
 const Book = () => {
-  (
-    <div className="container">
+  const books = useSelector((state) => state.booksReducer);
```

Comment:
- [ ] In order to get books from the state you should be using:

`  const allBooks = useSelector((state) => state.books);`

 `state.booksReducer` will return undefined.

Reply:
@ShoiraTa